totalRP3/Modules/Register/Main/RegisterTemplates.lua

```diff
@@ -10,3 +12,111 @@ end
 function TRP3_RegisterSectionHeaderMixin:SetIconTexture(icon)
 	self.Icon:SetIconTexture(icon);
 end
+
+TRP3_RegisterTraitLineMixin = {};
+
+function TRP3_RegisterTraitLineMixin:OnEnter()
+	TRP3_API.register.togglePsychoCountText(self, true);
+end
+
+function TRP3_RegisterTraitLineMixin:OnLeave()
+	TRP3_API.register.togglePsychoCountText(self, false);
+end
+
+function TRP3_RegisterTraitLineMixin:SetLeftText(text)
+	text = string.gsub(text or "", "%s+", " ");
+	self.LeftText:SetText(text);
+end
+
+function TRP3_RegisterTraitLineMixin:SetRightText(text)
+	text = string.gsub(text or "", "%s+", " ");
+	self.RightText:SetText(text);
+end
+
+function TRP3_RegisterTraitLineMixin:SetLeftIcon(icon)
+	self.RightIcon:SetIconTexture(icon);
```

Comment:
```suggestion
	self.LeftIcon:SetIconTexture(icon);
```
I reckon this is the culprit for the `?` issue.